daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -412,13 +412,15 @@ object Repl {
               )
             var count = 0
             val startTime = System.nanoTime()
+            var finished: SValue = null
             var errored = false
-            while (!machine.isFinal && !errored) {
+            while (finished == null && !errored) {
```

Comment:
This doesn’t make sense. You will never run more than one iteration of this loop.

Reply:
Yes. Thank you. Fixed